packages/react/src/themes/base/components/Icon/iconStyles.ts

```diff
@@ -61,27 +31,39 @@ const getPaddedStyle = (): ICSSInJSStyle => ({
 })
 
 const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
-  root: ({ props: p, variables: v, theme }): ICSSInJSStyle => {
-    const iconSpec = theme.icons[p.name]
-    const rtl = theme.rtl
-    const isFontBased = p.name && (!iconSpec || !iconSpec.isSvg)
+  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    speak: 'none',
+    verticalAlign: 'middle',
+
+    ...getXSpacingStyles(p.xSpacing, v.horizontalSpace),
+
+    ...(p.bordered && getBorderedStyles(v.borderColor)),
+    ...(p.circular && { ...getPaddedStyle(), borderRadius: '50%' }),
+    ...(p.disabled && {
+      color: v.disabledColor,
+    }),
+  }),
+  fontRoot: ({ props: p, variables: v, theme: t }): ICSSInJSStyle => {
+    const iconSpec = t.icons[p.name] || emptyIcon
```

Comment:
Does the base theme have font awesome icons?? And if it provides styles for font awesome, why not svg as well? We have support for font and svg icons, but base theme implements only font. This seems awkward to me..

Reply:
> Does the base theme have font awesome icons?

Nope.

> And if it provides styles for font awesome, why not svg as well? 

It provides styles for our required icons like `stardust-arrow-up` which are characters in Base theme.